api/database/migrations/2023_07_17_135053_add_screened_out_interest_responsiveness.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
```

Comment:
Unused imports

Reply:
Removed in [fix migration](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7296/commits/ce956e8f807c91f21443856633a539944f7fce8b)